beacon-chain/blockchain/receive_attestation.go

```diff
@@ -88,30 +93,47 @@ func (s *Service) spawnProcessAttestationsRoutine(stateFeed *event.Feed) {
 		}
 
 		st := slots.NewSlotTicker(s.genesisTime, params.BeaconConfig().SecondsPerSlot)
+		pat := slots.NewSlotTickerWithOffset(s.genesisTime, -reorgLateBlockCountAttestations, params.BeaconConfig().SecondsPerSlot)
 		for {
 			select {
 			case <-s.ctx.Done():
 				return
+			case <-pat.C():
+				root := s.UpdateHead(s.ctx)
+				if err := s.forkchoiceUpdateWithExecution(s.ctx, root, s.CurrentSlot()+1); err != nil {
```

Comment:
Using `s.CurrentSlot()+1` is correct here

Reply:
yes because this is called before the start of the slot, different than the other call